src/Cli/dotnet/Installer/Windows/DetectState.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Installer.Windows
+{
+    internal enum DetectState
```

Comment:
We should have a documentation link for all classes like this

Reply:
I added more documentation